### PR TITLE
Create footer

### PR DESCRIPTION
**Pull request by JuanequeX**
Description
---
This section contains links to the sections on the page,
information about contact and social media of brand.

### CODE REVIEW

File: app/javascript/components/footer/footer.scss [hunk 1]
```diff
@@ -0,0 +1,131 @@
+ @import '../.././styles/main.scss';
+
+.content {
+  grid-row-gap: 25px;
+  &__desktop {
```
Comment (OswaldoPineda) [inline app/javascript/components/footer/footer.scss:6]: add an enter here

File: app/javascript/components/footer/footer.scss [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    @media only screen and (min-width: 700px) {
```
Comment (OswaldoPineda) [inline app/javascript/components/footer/footer.scss:7]: use the variable for the breakpoint

File: app/javascript/components/footer/footer.scss [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+      padding: 0 28px;
+      width: 100%;
+   }
+  }
+}
+
+.footer__logo {
+width: 234px;
+height: 51px;
+}
```
Comment (OswaldoPineda) [inline app/javascript/components/footer/footer.scss]: check the indentation here, and sort alphabetically

File: app/javascript/components/footer/footer.scss [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+.footer {
+  background: $black;
+  color: $white;
+  display: flex;
+  justify-content: center;
+  margin: 24px 0 0;
+  padding: 18px 0 7px;
+
+  &__logo-anchor {
+    @media only screen and (min-width: 700px) {
```
Comment (OswaldoPineda) [inline app/javascript/components/footer/footer.scss]: use the variable here

File: app/javascript/components/footer/footer.scss [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  &__information-list-child {
+      a {
+        color: $white;
+        font-family: $font-stack;
+        font-size: 16px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 1;
+        text-decoration: none;
+      }
```
Comment (OswaldoPineda) [inline app/javascript/components/footer/footer.scss:67]: add an extra line here

File: app/javascript/components/footer/footer.scss [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+      &:nth-child(2) {
```
Comment (OswaldoPineda) [inline app/javascript/components/footer/footer.scss:69]: here

File: app/javascript/components/footer/footer.scss [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+        margin: 15px 0px;
+      }
+      &:nth-child(3) {
```
Comment (OswaldoPineda) [inline app/javascript/components/footer/footer.scss:73]: and here

File: app/javascript/components/footer/footer.scss [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+  &__contact-list-child {
+    a {
+      color: $white;
+      font-family: $font-stack;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 1;
+      text-decoration: none;
+    }
```
Comment (OswaldoPineda) [inline app/javascript/components/footer/footer.scss:99]: and here

File: app/javascript/components/footer/footer.scss [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+  }
+&__copyright {
+    color: $white;
+  font-family: $font-stack;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 1;
+  text-decoration: none;
```
Comment (OswaldoPineda) [inline app/javascript/components/footer/footer.scss]: check the indentation here

**Review by OswaldoPineda — CHANGES_REQUESTED** [review]
nice, only some comments

File: app/javascript/components/footer/footer.scss (hunk 1 above)
Comment (JuanequeX) [reply app/javascript/components/footer/footer.scss:6]: done

File: app/javascript/components/footer/footer.scss (hunk 2 above)
Comment (JuanequeX) [reply app/javascript/components/footer/footer.scss:7]: done

File: app/javascript/components/footer/footer.scss (hunk 3 above)
Comment (JuanequeX) [reply app/javascript/components/footer/footer.scss]: done

File: app/javascript/components/footer/footer.scss (hunk 4 above)
Comment (JuanequeX) [reply app/javascript/components/footer/footer.scss]: done

File: app/javascript/components/footer/footer.scss (hunk 5 above)
Comment (JuanequeX) [reply app/javascript/components/footer/footer.scss:67]: done

File: app/javascript/components/footer/footer.scss (hunk 6 above)
Comment (JuanequeX) [reply app/javascript/components/footer/footer.scss:69]: done

File: app/javascript/components/footer/footer.scss (hunk 8 above)
Comment (JuanequeX) [reply app/javascript/components/footer/footer.scss:99]: done

File: app/javascript/components/footer/footer.scss (hunk 7 above)
Comment (JuanequeX) [reply app/javascript/components/footer/footer.scss:73]: done

File: app/javascript/components/footer/footer.scss (hunk 9 above)
Comment (JuanequeX) [reply app/javascript/components/footer/footer.scss]: done

**Review by OswaldoPineda — APPROVED** [review]

**Pull request merged**